File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/mad/Java/Go/s684030429_commented.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

type Main struct {
	sc  *bufio.Scanner
	max int
	cur int
}

func (m *Main) solve() {
	a := m.ni()
	b := m.ni()

	fmt.Println(max(0, a-2*b))
}

func max(x, y int) int {
	if x > y {
		return x
	}
	return y
}

func (m *Main) ni() int {
	m.sc.Scan()
	num, _ := strconv.Atoi(m.sc.Text())
	return num
}

func (m *Main) nl() int64 {
	m.sc.Scan()
	num, _ := strconv.ParseInt(m.sc.Text(), 10, 64)
	return num
}

func (m *Main) nf() float32 {
	m.sc.Scan()
	num, _ := strconv.ParseFloat(m.sc.Text(), 32)
	return float32(num)
}

func (m *Main) nd() float64 {
	m.sc.Scan()
	num, _ := strconv.ParseFloat(m.sc.Text(), 64)
	return num
}

func (m *Main) ns() string {
	m.sc.Scan()
	return m.sc.Text()
}

func (m *Main) nst(s string) []string {
	return strings.Fields(s)
}

func (m *Main) ia(a []int) {
	for i := range a {
		a[i] = m.ni()
	}
}

func (m *Main) la(a []int64) {
	for i := range a {
		a[i] = m.nl()
	}
}

func (m *Main) fa(a []float32) {
	for i := range a {
		a[i] = m.nf()
	}
}

func (m *Main) da(a []float64) {
	for i := range a {
		a[i] = m.nd()
	}
}

func (m *Main) sa(a []string, empty bool) {
	if empty {
		for i := range a {
			a[i] = ""
		}
	} else {
		for i := range a {
			a[i] = m.ns()
		}
	}
}

func (m *Main) ida(a [][]int, n, m int) {
	for i := 0; i < n; i++ {
		for j := 0; j < m; j++ {
			a[i][j] = m.ni()
		}
	}
}

func (m *Main) lda(a [][]int64, n, m int) {
	for i := 0; i < n; i++ {
		for j := 0; j < m; j++ {
			a[i][j] = m.nl()
		}
	}
}

func (m *Main) dda(a [][]float64, n, m int) {
	for i := 0; i < n; i++ {
		for j := 0; j < m; j++ {
			a[i][j] = m.nd()
		}
	}
}

func (m *Main) stoi(s string) int {
	num, _ := strconv.Atoi(s)
	return num
}

func (m *Main) stod(s string) float64 {
	num, _ := strconv.ParseFloat(s, 64)
	return num
}

func (m *Main) lmin(a []int64) int64 {
	min := int64(^uint64(0) >> 1)
	for _, v := range a {
		if v < min {
			min = v
		}
	}
	return min
}

func (m *Main) imin(a []int) int {
	min := int(^uint(0) >> 1)
	for _, v := range a {
		if v < min {
			min = v
		}
	}
	return min
}

func (m *Main) lmax(a []int64) int64 {
	max := int64(^uint64(0) >> 1) * -1
	for _, v := range a {
		if v > max {
			max = v
		}
	}
	return max
}

func (m *Main) imax(a []int) int {
	max := int(^uint(0) >> 1) * -1
	for _, v := range a {
		if v > max {
			max = v
		}
	}
	return max
}

func (m *Main) ibs(a []int, toFind int) bool {
	// Assuming the array is already sorted
	min, max := 0, len(a)-1
	for min <= max {
		mid := min + (max-min)/2
		if a[mid] == toFind {
			return true
		} else if toFind > a[mid] {
			min = mid + 1
		} else {
			max = mid - 1
		}
	}
	return false
}

func (m *Main) lbs(a []int64, toFind int64) bool {
	// Assuming the array is already sorted
	min, max := 0, len(a)-1
	for min <= max {
		mid := min + (max-min)/2
		if a[mid] == toFind {
			return true
		} else if toFind > a[mid] {
			min = mid + 1
		} else {
			max = mid - 1
		}
	}
	return false
}

func (m *Main) stb(s string) int {
	sum := 0
	k := 0
	for i := len(s) - 1; i >= 0; i-- {
		num, _ := strconv.Atoi(string(s[i]))
		sum += num * (1 << k)
		k++
	}
	return sum
}

func main() {
	sc := bufio.NewScanner(os.Stdin)
	sc.Split(bufio.ScanWords)
	m := Main{sc: sc}
	m.solve()
}

// End of Code

File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/mad/Java/Python/s684030429_commented.py
import sys

class Main:
    def __init__(self):
        self.sc = sys.stdin

    def solve(self):
        a = self.ni()
        b = self.ni()
        print(max(0, a - 2 * b))

    def ni(self):
        return int(self.sc.readline().strip())

    def nl(self):
        return int(self.sc.readline().strip())

    def nf(self):
        return float(self.sc.readline().strip())

    def nd(self):
        return float(self.sc.readline().strip())

    def ns(self):
        return self.sc.readline().strip()

    def ia(self, n):
        return [self.ni() for _ in range(n)]

    def la(self, n):
        return [self.nl() for _ in range(n)]

    def fa(self, n):
        return [self.nf() for _ in range(n)]

    def da(self, n):
        return [self.nd() for _ in range(n)]

    def sa(self, n, empty=False):
        if empty:
            return [""] * n
        else:
            return [self.ns() for _ in range(n)]

    def ida(self, n, m):
        return [self.ia(m) for _ in range(n)]

    def lda(self, n, m):
        return [self.la(m) for _ in range(n)]

    def dda(self, n, m):
        return [self.da(m) for _ in range(n)]

    def stoi(self, s):
        return int(s)

    def stod(self, s):
        return float(s)

    def lmin(self, a):
        return min(a)

    def imin(self, a):
        return min(a)

    def lmax(self, a):
        return max(a)

    def imax(self, a):
        return max(a)

    def ibs(self, a, toFind):
        a.sort()
        min_index = 0
        max_index = len(a) - 1
        found = False

        while min_index <= max_index and not found:
            mid = min_index + (max_index - min_index) // 2
            if a[mid] == toFind:
                found = True
            elif toFind > a[mid]:
                min_index = mid + 1
            else:
                max_index = mid - 1

        return found

    def lbs(self, a, toFind):
        a.sort()
        min_index = 0
        max_index = len(a) - 1
        found = False

        while min_index <= max_index and not found:
            mid = min_index + (max_index - min_index) // 2
            if a[mid] == toFind:
                found = True
            elif toFind > a[mid]:
                min_index = mid + 1
            else:
                max_index = mid - 1

        return found

    def stb(self, s):
        return int(s, 2)

if __name__ == "__main__":
    main = Main()
    main.solve()
# End of Code
